scanner/cmd/update.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"context"
+
+	"github.com/quay/zlog"
+	"github.com/stackrox/scanner/v4/updater"
+)
+
+func main() {
```

Comment:
I recommend putting this in a separate directory from the `scanner.go` file. I'd put this in something like `scanner/cmd/updater/updater.go` and the other in `scanner/cmd/scanner/scanner.go`. This way it is clear that these are two separate main functions